qiskit/assembler/assemble_schedules.py

```diff
@@ -30,6 +30,7 @@
 from qiskit.qobj.utils import MeasLevel, MeasReturnType
 
 from .run_config import RunConfig
+from ..pulse.transforms import compress_pulses
```

Comment:
Perhaps we could use the absolute import here to make it more clear where this is coming from? I would prefer something like `from qiskit.pulse import transforms` `transforms.compress_pulses` but I understand Qiskit does use relative imports in a lot of places.